### ISSUE TITLE
there a loss value?

### ISSUE THREAD

**Issue by farissetiawan6** [remicres/otbtf#83]
Hello, I want to ask. Is there a loss training and validation value in implementing deep learning CNN with otbtf? Is it just precision, recall, f-score and accuracy? Thanks

**Comment by remicres** [comment]
Hello @farissetiawan6 

The loss value is implemented in the network graph. For instance if you take a look in [this example](https://github.com/remicres/otbtf/blob/develop/otbtf/examples/tensorflow_v1x/create_savedmodel_simple_cnn.py#L69) you can see the loss.

Evaluation metrics are another thing. 
In OTBTF application, the usual classification metrics are implemented (precision, recall, accuracy for each class, for all, kappa, etc.).

**Comment by farissetiawan6** [comment]
How can I see the network graph?

**Comment by remicres** [comment]
The graph is a concept in TensorFlow. You can check the TensorFlow documentation for more info
